docs(scalers): Update `cpu` and `memory` scaler docs for validating `requests`/`limits` from `LimitRange` (#1296)
kedacore/keda-docs: content/docs/2.13/scalers/cpu.md

> KEDA uses standard `cpu` and `memory` metrics from the Kubernetes Metrics Server, which is not installed by default on certain Kubernetes deployments such as EKS on AWS. Additionally, the `resources` section of the relevant Kubernetes Pods must include `requests` (at a minimum).

Reviewer: nit: we may as well fix this sentence, at least that is my observation testing the webhook from https://github.com/kedacore/keda/pull/5377/
```suggestion
KEDA uses standard `cpu` and `memory` metrics from the Kubernetes Metrics Server, which is not installed by default on certain Kubernetes deployments such as EKS on AWS. Additionally, the `resources` section of the relevant Kubernetes Pods must include at least one of `requests` or `limits`.
```

Author: My apologies @wozniakjan! I was busy with something. Thank you and @zroubalik for adding the commit in. :blush:
